build-fixer.mjs

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import fs from "fs";
+const args = process.argv;
+const command = args[2];
+import { exec } from "child_process";
+
+
+if (!command) {
+  console.log("No command specified");
+  process.exit(1);
+}
+
+function deleteHelper(path) {
+
+  if (fs.existsSync(path)) {
+    const lstat = fs.lstatSync(path);
+    if (lstat.isFile()) {
+      fs.unlinkSync(path);
+    } else {
+      const files = fs.readdirSync(path);
+      files.forEach(file => {
+        deleteHelper(`${path}/${file}`);
+      });
+      fs.rmdirSync(path);
+    }
+  }
+}
+
+let commandResult;
+const env = {
+  NODE_ENV: "production",
+};
+
+switch (command) {
+  case "clean":
+    deleteHelper("dist");
+    break;
+  case "build":
+
+    switch (process.platform) {
+      case "win32":
+        commandResult = "npx rollup -c";
+        break;
+
+      default:
+        commandResult = "NODE_ENV=production npx  rollup -c";
+        break;
+    }
+
+    exec(commandResult, {...process.env, env }, (_error, _stdout, _stderr) => {
```

Comment:
I tested this code, and fail in linux and throw `npx not found`. So I think  we should remove {...process.env, env} for exec arguments and replace it with inline env. Something like this
```js
switch (process.platform) {
      case "win32":
  //note that production&& must not separate with space
        commandResult = "set NODE_ENV=production&& npx rollup -c";
        break;

      default:
        commandResult = "NODE_ENV=production npx  rollup -c";
        break;
    }

exec(commandResult, (_error, _stdout, _stderr)
```